provisioning/miniprov/hare_mp/store.py

```diff
@@ -22,11 +22,15 @@
 from typing import Any
 
 from cortx.utils.conf_store import ConfStore
+from hare_mp.types import MissingKey
 
 
 class ValueProvider:
     def get(self, key: str) -> Any:
-        return self._raw_get(key)
+        value = self._raw_get(key)
```

Comment:
Do you unit tests pass for you?

Reply:
It was failing. So I have modified test case.
@knekrasov Please have a look